Services/Init/classes/class.ilInitialisation.php

```diff
@@ -1578,7 +1580,8 @@ protected static function initHTML() : void
             // or not set at all (then we want the last offset, e.g. being used from a session var).
             // So I added the wrapping if statement. Seems to work (hopefully).
             // Alex April 14th 2006
-            if (isset($_GET['offset']) && $_GET['offset'] != "") {                            // added April 14th 2006
+            // @todo not replaced by refinery due to unknown sideeffects
```

Comment:
I wondered what "unknown sideeffect" means. What is unknown about them?

Reply:
The side effects in my opinion are that we don't know, if somebody deep down in the calling chain might still need the value added to the $_GET. Thus changing this wo only read it with refinery would break the code. Changing this requires a scan through the entire code base.